### PR TITLE
Hides the media controls and scoreboard when screen width < 768px

### PR DESCRIPTION
**Pull request by brandonb927**
Also tweaks the media controls to look less like buttons, and moves the turn count to below the board

kapture 2019-02-13 at 19 01 55

### CODE REVIEW

File: src/theme/index.js [hunk 1]
```diff
@@ -46,6 +46,13 @@ export const colors = {
   // purple: "#240458",
 };
 
+export const breakpoints = {
```
Comment (brandonb927) [inline src/theme/index.js]: These breakpoints match the Bootstrap 4 ones used in Play

File: src/reducers/index.js [hunk 2]
```diff
@@ -32,28 +38,25 @@ export default (state = {}, action) => {
...
-      {
-        action: "HIGHLIGHT_SNAKE",
-        id: action.snakeId,
-        name: action.snakeId
-          ? state.currentFrame.snakes.find(s => s._id === action.snakeId).name
-          : null
-      },
-      "*"
-    );
+    window.parent.postMessage(data, "*");
```
Comment (brandonb927) [inline src/reducers/index.js]: Made this a generic postMessage function now

File: src/reducers/index.js [hunk 3]
```diff
@@ -17,6 +17,12 @@ export default (state = {}, action) => {
       return { ...state, gameStatus: action.status };
     case "SET_CURRENT_FRAME":
       return { ...state, currentFrame: action.frame };
+    case "SET_TURN_COUNT":
+      windowPostMessage({
```
Comment (brandonb927) [inline src/reducers/index.js]: Following @ColDog pattern with posting whats is captured in the redux action via `postMessage`

File: src/components/scoreboard.jsx [hunk 4]
```diff
@@ -26,11 +28,18 @@ const orderSnakes = snakes => {
...
-}));
+const Wrapper = styled("div")`
+  display: none;
+  padding-right: 2rem;
+  color: ${({ theme }) =>
+    theme === themes.dark ? colors.lightText : colors.darkText};
+  font-weight: 700;
+  font-size: 2.2rem;
+
+  @media (min-width: ${breakpoints.md}) {
```
Comment (brandonb927) [inline src/components/scoreboard.jsx]: An example of using the medium breakpoint to show/hide an element while also retaining the query string param hide/show method as well

File: src/reducers/index.js (hunk 3 above)
Comment (colin-swu) [reply src/reducers/index.js]: Nice, I was thinking about this we should probably make it redux middleware at some point in the future, I think that's a better separation of concerns. It seems fine now though!

**Pull request merged**